app/models/solidus/gateway/braintree_gateway.rb

```diff
@@ -7,6 +7,7 @@ class Gateway::BraintreeGateway < ::Spree::Gateway
     preference :public_key, :string
     preference :private_key, :string
     preference :always_send_bill_address, :boolean, default: false
+    preference :verify_shipping_address, :boolean, default: true
```

Comment:
There is also the `always_send_bill_address` preference, which seems to serve the same purpose. Perhaps we should follow that naming convention? Although, I must admit, I don't care for it.